src/pubsub.c

```diff
@@ -271,18 +303,25 @@ int pubsubSubscribeChannel(client *c, robj *channel, pubsubtype type) {
 /* Unsubscribe a client from a channel. Returns 1 if the operation succeeded, or
  * 0 if the client was not subscribed to the specified channel. */
 int pubsubUnsubscribeChannel(client *c, robj *channel, int notify, pubsubtype type) {
+    dict *d;
     dictEntry *de;
     list *clients;
     listNode *ln;
     int retval = 0;
+    unsigned int slot = 0;
```

Comment:
nit: `unsigned slot = 0;`

Reply:
Do we have a policy to prefer `unsigned` over `unsigned int`?

Reply:
@oranagra recommended in one of my PR. I'm fine either way.

Reply:
I don't feel necessary to change this ... and we already have hundreds of `unsigned int` in Redis.

Reply:
> @oranagra recommended in one of my PR. I'm fine either way.

i don't recall that.
AFAICT redis usually uses `unsigned int`, let's keep it.